client/termdb/TermTypeSearch.ts

```diff
@@ -109,6 +109,12 @@ export class TermTypeSearch {
 	async addTabsAllowed(state) {
 		for (const type of this.types) {
 			const termTypeGroup = typeGroup[type]
+			let label = termTypeGroup
+			if (type == TermTypes.GENE_VARIANT) {
+				label = 'Mutation'
+				if (this.app.vocabApi.termdbConfig.queries.cnv) label = 'Mutation/CNV'
+				if (this.app.vocabApi.termdbConfig.queries.fusion) label = label + '/Fusion'
```

Comment:
it's `queries.svfusion` not fusion

Reply:
const labels=[]
if(xx) labels.push('Mutation')
if(yy) ...
return labels.join('/')

Reply:
Done